app/controllers/HomeController.php

```diff
@@ -3,7 +3,10 @@
 use LaravelDuo\LaravelDuo;
 use Artdarek\OAuth\Facade\OAuth;
 
-class HomeController extends BaseController {
+class HomeController extends BaseController
+{
+    // 6 hours
+    const MAX_EXPIRY = 21600;
```

Comment:
Is there any particular reason for this?

A 24 hour lease is helpful when we are WFH.